### DISCUSSION TITLE
I want to use the same key double push (Triple times) as a hotkey to ctrl ctrl, etc.

### DISCUSSION THREAD

**Discussion by molio** [Flow-Launcher/Flow.Launcher#954] [Q&A]
Others, such as ShiftShiftShift.

**Comment by onesounds** [comment]
- Double ctrl is in our plan because it is the basic shortcut to the listary
- I only thought about Ctrl. Do we need Shift too? I think it's possible because the function is different from alt.
- I'm not sure three times is an appropriate implementation. If there is a program that works like this, please let me know.

**Reply by molio** [reply]
> * Double ctrl is in our plan because it is the basic shortcut to the listary

I hope it will be implemented as soon as possible.

> * I only thought about Ctrl. Do we need Shift too? I think it's possible because the function is different from alt.

 For example, If ctrl is already used for the clipboard tool, is it not necessary?

> * I'm not sure three times is an appropriate implementation. If there is a program that works like this, please let me know.

I can't remember right away, but I remember that there was a thing that could be set in the copy and paste tool.
If you think it's inappropriate, though, I don't mind I can long press and hold.

https://itenium.be/blog/productivity/autohotkey-hotkeys/#double-triple-and-long-key-presses
https://www.autohotkey.com/board/topic/98713-how-to-differentiate-double-and-triple-press/
Please don't tell me to use AHK.

Best regards.

**Comment by HamzaETTH** [comment]
any progress on this? I am simply too used to Listary shortcut and would rather not use an AutoHotkey script.

**Reply by jjw24** [reply]
I am working on a new hotkey API, this is on my roadmap.

**Reply by Seryta** [reply]
Any progress?
Best regards.

**Reply by Seryta** [reply]
Double‑tapping Ctrl is really essential. Like you said, I’ve gotten so used to Listary’s shortcuts—and it’s simpler, isn’t it?
